### ISSUE TITLE
Missing cluster examples

### ISSUE THREAD

**Issue by timrobertson100** [gbif/pipelines#546]
This issue serves as a placeholder to capture occurrences that could or should cluster, but don't.

1. https://www.gbif.org/occurrence/2418710210 and https://www.gbif.org/occurrence/2829590452
    a. Note the field number format, where one is prefixed with the initials of the collector
    b. Note that one has been re-identified

**Comment by jholetschek** [comment]
- Taxa are different
- All identfiers are different: CatalogNumber, occurrenceID, recordNumber (added prefix), otherCatalogNumbers
- Institution and collections codes are different

**Comment by MortenHofft** [comment]
Another example 
cluster of 
* [paratype from NHM london](https://www.gbif.org/occurrence/1826086349) and 
* [treatment](https://www.gbif.org/occurrence/2461888270) 
 
is missing the [sequence from EMBL](https://www.gbif.org/occurrence/3346047803)

**Comment by jholetschek** [comment]
Yes, we're currently working on that. Similar as this one: https://github.com/pensoft/BiCIKL/issues/29
Upon importing from EMBL, the catalognumber is preceeded with institution code. But BMNH isn't the original instituion code, so that's a bit mysterious.
